schematic/manifest/generator.py

```diff
@@ -1555,33 +1552,25 @@ def get_manifest(
             if self.use_annotations:
                 if self.is_file_based:
                     annotations = store.getDatasetAnnotations(dataset_id)
-                    # in the old code, we take a subset of columns if no interested in user-defined annotations and there are files present
-
-                # if there are no files with annotations just generate an empty manifest
-                if annotations.empty:
```

Comment:
I changed it so that `get_dataframe_by_url ` do not get called when use_annotations is set to True and is_file_base is also True